Jenkinsfile

```diff
@@ -54,17 +54,16 @@ def robotest() {
       propagateParamsToEnv()
     }
   }
-  runRobotest = (env.RUN_ROBOTEST == 'run')
```

Comment:
Is this just a guess since I thought booleans were [serializable](https://www.jenkins.io/blog/2017/02/01/pipeline-scalability-best-practice/#appendix-serializable-vs.-non-serializable-types) in groovy?

Reply:
Yeah, this was a hunch.  Looks like it didn't pan out:

https://jenkins.gravitational.io/job/Gravity-PR-5.2/job/PR-2223/5/console